src/extension/telemetry.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import TelemetryReporter from 'vscode-extension-telemetry';
+import { publisher, name, version } from '../../package.json';
+
+let reporter: TelemetryReporter;
+
+export function activate() {
+    const key = 'bf8e52c4-6749-4709-92a0-e3a8fd589648';
```

Comment:
should we have a seperate config to store keys like these? I'm assuming so far we only work with 1 so seperate file might be overkill?

Reply:
There's no intention of ever changing/configuring this, so I think it should be ok.

Note https://github.com/microsoft/vscode-cpptools and https://github.com/microsoft/sarif-website do the same.

Reply:
Sounds good, thanks